dbatools.psd1

```diff
@@ -544,6 +544,7 @@
         # solvable filesystem issues or other workarounds
         'Install-DbaSqlWatch',
         'Uninstall-DbaSqlWatch',
+        'Remove-DbaDbView',
```

Comment:
Please remove this from the PR, maintainer control this changes to the manifest and psm1 files.

Reply:
Sorry, I must have clicked the wrong button :)